tophatmonocle/hivemind-service
# Created authenticated users

- Added authentication to all api calls

DevQA
- run `inv docker.migrate`
- run `inv docker.exec`
- run `manage.py createsuperuser --username [your username eg. owenchak] --email [your tophat email]`
- In postman, make a POST request to http://127.0.0.1:8888/api-token-auth with body>form data selected and enter your username/password as shown below
![image](https://user-images.githubusercontent.com/20757300/61059570-3684c100-a3c7-11e9-8b96-b06012f59fec.png)

You should expect to see the api return a valid token

- Make a GET request to http://127.0.0.1:8888/linkedin_profile_data/match?li_url=http://linkedin.com
- You should expect to see an authentication error
- Add a header key=Authorization and value=Token {your token we previously fetched}
- Should expect a valid api response with a status of 200
